src/AppContext.tsx

```diff
@@ -1,23 +1,23 @@
-import React from 'react';
+import React from "react";
 
 export type Facility = {
-  id: number,
-  facilityName: string
-}
+  id: number;
+  facilityName: string;
+};
 
 export type Quarter = {
-  id: number,
-  year: number,
-  quarter: number
-}
+  id: number;
+  year: number;
+  quarter: number;
+};
 
 export type AppContextType = {
-  facilities: Facility[],
-  quarters: Quarter[]
-}
+  // facilities: Facility[], RM
```

Comment:
Just as an FYI, put @ TODO's so that you do not forget in the near future

Reply:
don't merge, I haven't tested with the backend.